Sources/NSTimer+ClosureKit.swift

```diff
@@ -24,8 +24,8 @@ final class BlockTimer: NSObject {
     class func scheduledTimer(withTimeInterval interval: TimeInterval, repeated: Bool = false,
         handler: @escaping CKTimerHandler) -> Timer
     {
-        return Timer.scheduledTimer(timeInterval: interval, target: self,
-            selector: #selector(BlockTimer.invokeFrom(timer:)),
+        return .scheduledTimer(timeInterval: interval, target: self,
+            selector: #selector(invokeFrom),
```

Comment:
I intentionally use the full selector names to avoid ambiguities when reading it.